CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Running multiple competitions at the same time with own deadlines
+- Sorting result tables with jQuery tablesorter 2.0
```

Comment:
@snukky: specifically mention that we opt for `MIT` license here?

Reply:
Yes, it's double licensed, so this totally makes sense.